TechnicalDocuments/NewHiresCheckList.md

```diff
@@ -60,6 +60,8 @@ Here's how to get the iOS project up and running.
      git clone git@github.com:Babylonpartners/babylon-ios.git
      ```
 
+1. Get our latest Xcode templates by running `./Templates/install_xcode_templates.sh`
```

Comment:
we also have some git hooks in `githooks` folder

Reply:
I've started to work on a hook to install updates to the templates automatically btw

Reply:
We do?